packages/mongo-nav/src/org-nav/GovBanner.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import GovBuildingIcon from '@leafygreen-ui/icon/dist/GovBuilding';
+import Tooltip from '@leafygreen-ui/tooltip';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { Body } from '@leafygreen-ui/typography';
+
+const desktopContainer = css`
+  margin-bottom: 10px;
+  position: relative;
+  z-index: 2;
+  display: flex;
+  flex-grow: 1;
+  height: 5px;
+  background-color: #edf5fa;
+  align-items: flex-start;
+  justify-content: center;
+`;
+
+const tabletContainer = css`
+  position: relative;
+  z-index: 2;
+  display: flex;
+  flex-grow: 1;
+  align-items: flex-start;
+  justify-content: center;
+  height: 0;
+`;
+
+const mobileContainer = css`
+  width: 30px;
+  height: 30px;
+  background-color: #edf5fa;
+  margin-left: 12px;
+  border-radius: 7px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+`;
+
+const sharedBannerContainerStyles = css`
+  background-color: #edf5fa;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  position: relative;
+`;
+
+const desktopTooltipContainer = css`
+  height: 34px;
+  width: 205px;
+  border-radius: 0 0 17px 17px;
```

Comment:
This border radius seems too small compared to the mocks. In fact, to match the mocks it looks like you need to maximize the border radius, so you can specify an arbitrarily large value (`50px`/`100px`) instead of `17px` here.

Reply:
That said, it looks like there's a mismatch of the border radius in the mocks, vs in the specs Nelle sent out via Slack. Might be worth double-checking which is correct.

Reply:
Fun tip: I believe `border-radius: 0 0 17px 17px;` is equivalent to `border-radius: 0 17px;` (though I may have that mixed up a bit 😅 )

Reply:
I QA'd the border radiuses with @nellem23 this morning, but happy to update if this doesn't look right to you.

Reply:
`0 17px` does not work in this case :/

Reply:
Could you just clarify with her? The border radius between the specs she sent today to us both are different than in her mocks.

Reply:
Ok just met with @nellem23 and bumped to 100px